### ISSUE TITLE
HttpException (HttpException: Connection closed while receiving data, uri = https://xxxxxxxxx.auth0.com/userinfo)

### ISSUE THREAD

**Issue by sawankumarbundelkhandi** [devdennysegura/flutter-auth0#22]
I am frequently receiving this error when calling the getUserInfo in Auth0Auth class.

![image](https://user-images.githubusercontent.com/10863510/65059399-95505300-d993-11e9-91b7-fcdce779adb6.png)

**Comment by devdennysegura** [comment]
can you try again with news changes?

**Comment by sawankumarbundelkhandi** [comment]
@devdennysegura No, It's still failing. I was testing the app with my fork only. I don't see any commit after the my fork is merged.

**Comment by shanazainab** [comment]
I tried the example code and getting this error
![Capture](https://user-images.githubusercontent.com/39184339/65811829-cd4d6680-e1db-11e9-8e0e-2b579a4db5e5.PNG)

**Comment by spencerreeves** [comment]
Has anyone made any headway on this issue?

**Comment by sawankumarbundelkhandi** [comment]
@spencerreeves I have workaround it for now. Check my code below

https://github.com/sawankumarbundelkhandi/flutter-auth0/commit/ad55fc794aec6af1d22e1825b1b5fe85b5332d96
